Installers/default.build

```diff
@@ -106,10 +107,11 @@
          </fileset>
       </copy>
       <copy todir="root/Library/Frameworks/MonoGame.framework/v${buildNumber}/Assemblies/DesktopGL">
-         <fileset basedir="../MonoGame.Framework/bin/DesktopGL/AnyCPU/Release">
+         <fileset basedir="../MonoGame.Framework/bin/Linux/AnyCPU/Release">
```

Comment:
It was.. Currently we do not build a "DesktopGL" platform we only build Linux and WindowsGL. As a result we currently get no Assemblies in that directory in the installer even though we have projects trying to reference them. The Windows Installer uses the WindowsGL binaries for that folder, we should do the same and Linux=WindowsGL now.